### PR TITLE
Add directives

### PR DESCRIPTION
**Pull request by SebastianGode**

### CODE REVIEW

**Comment by SebastianGode** [comment]
recheck

File: doc/requirements.txt [hunk 1]
```diff
@@ -6,3 +6,5 @@
 
 otc-api-ref>=0.1.0 # Apache-2.0
 reno>=3.1.0 # Apache-2.0
+git+https://github.com/opentelekomcloud/otc-sphinx-directives.git@error_fix#egg=otc_sphinx_directives
```
Comment (gtema) [inline doc/requirements.txt:9]: please never never never do that in a project like that

File: doc/requirements.txt [hunk 2]
```diff
@@ -6,3 +6,5 @@
...
+git+https://gitea.eco.tsi-dev.otc-service.com/infra/otc-metadata.git#egg=otc_metadata
```
Comment (gtema) [inline doc/requirements.txt:10]: this repo should really NEVER get such dependency

**Review by gtema — CHANGES_REQUESTED** [review]

File: doc/requirements.txt [hunk 3]
```diff
@@ -6,3 +6,5 @@
 
 otc-api-ref>=0.1.0 # Apache-2.0
 reno>=3.1.0 # Apache-2.0
+git+https://github.com/opentelekomcloud/otc-sphinx-directives.git#egg=otc_sphinx_directives
```
[the remark this answers is not in the document]
Comment (tischrei) [reply doc/requirements.txt]: That's not a "real PR" we wanna see how the behavior is.

[the remark this answers is not in the document]
Comment (tischrei) [reply doc/requirements.txt]: I converted it to draft to make it clear.

**Comment by tischrei** [comment]
not needed anymore

**Pull request closed without merging**